arrow/arrow.py

```diff
@@ -229,7 +227,9 @@ def utcfromtimestamp(cls, timestamp):
         )
 
     @classmethod
-    def fromdatetime(cls, dt, tzinfo=None):
+    def fromdatetime(
+        cls, dt: Union[dt_datetime, "Arrow"], tzinfo: Any = None
```

Comment:
Should `tzinfo` be of type `Optional[Any]` since None is an appropriate value to pass in if a user does not choose to use the default argument?